src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,37 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" if:true={isLoading}>
+                <lightning-spinner alternative-text="Loading"></lightning-spinner>
+            </div>
+            <div class="slds-grid slds-grid_align-center slds-grid_vertical">
+                <div class="slds-col slds-align_absolute-center">
+                    <img class="image" src={img}/>
```

Comment:
alt text or other accessibility attribute is required here

Reply:
If the image is decorative only and has no critical meaning, it can have an empty alt tag (`alt=""`) to tell the screen reader to skip it. Otherwise the screen reader will try to read it and it will sound like "Image <filename>" or something like that.

If the image *does* have meaning to the user, the alt text needs to describe it.